Applications/web_portal/web_app/tests/login_page_tester/source/index.tsx

```diff
@@ -1,8 +1,60 @@
+import {css, StyleSheet} from 'aphrodite';
+import * as Beam from 'beam';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import * as WebPortal from 'web_portal';
 
+/** The React state for the TestApp. */
+export interface State {
+  rejectMessage: string;
+}
+
+/** The React properties for the TestApp. */
+export interface Properties {
+
+}
+
 const model = new WebPortal.LocalLoginPageModel();
+const onLogin = (account: Beam.DirectoryEntry) => {
```

Comment:
Perhaps for the test, perform a console.log displaying the account's name within the onLogin.

Also make onLogin a proper function rather than a const variable, ie function onLogin(...).